### PR TITLE
Fixed BackgroundColor on SplashScreen winUI

### PR DESCRIPTION
**Pull request by pictos**
Fixes: #128

### CODE REVIEW

File: src/Resizetizer/src/GeneratePackageAppxManifest.cs [hunk 1]
```diff
@@ -377,6 +377,20 @@ void UpdateManifest(XDocument appx)
...
+					// BackgroundColor=""
+					{
+						var xname = "BackgroundColor";
+						var attr = visual.Element(xsplash);
+
+						if (attr is null || string.IsNullOrEmpty(attr.Value))
+						{
+							if (splashInfo?.Color is not null)
+							{
+								splash.SetAttributeValue(xname, Utils.SkiaColorWithoutAlpha(splashInfo.Color));
```
Comment (jeromelaban) [inline src/Resizetizer/src/GeneratePackageAppxManifest.cs:390]: Were there unit tests for this kind of change? Or were they only for mobile targets?

Comment (pictos) [reply src/Resizetizer/src/GeneratePackageAppxManifest.cs:390]: I'll update the unit tests. This change is for platforms that use `Package.manifest` file, windows, and Skia targets.

File: src/Resizetizer/src/GeneratePackageAppxManifest.cs [hunk 2]
```diff
@@ -377,6 +377,20 @@ void UpdateManifest(XDocument appx)
 							splash.SetAttributeValue(xname, path);
 						}
 					}
+
+					// BackgroundColor=""
```
Comment (nickrandolph) [inline src/Resizetizer/src/GeneratePackageAppxManifest.cs:381]: You need to remove the old code that sets the wrong BackgroundColor - https://github.com/unoplatform/uno.resizetizer/blob/34e99577d5fe3cfac192d158b5ce6b3f147af0db/src/Resizetizer/src/GeneratePackageAppxManifest.cs#L234

Comment (pictos) [reply src/Resizetizer/src/GeneratePackageAppxManifest.cs:377]: as discussed on teams, we will use the `UnoIcon.Background` value here

**Review by nickrandolph — APPROVED** [review]

**Pull request merged**